st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -348,35 +426,35 @@ def encrypt_archive(archive_file_path, debug=False):
     encrypted_archive_output_file_path = archive_file_path + '.asc'
```

Comment:
That's fine for now. Would be nice to get this all into a single file if possible. Not sure if anyone else has opinions on this?